internal/services/apimanagement/testdata/api_management_api_openapi_invalide.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
Similar typo in the file name? Should this be:
`internal/services/apimanagement/testdata/api_management_api_openapi_invalid.yaml` ?